### ISSUE TITLE
go to smooth (not bin averaged) functions for truth and likelihood

### ISSUE THREAD

**Issue by davidwhogg** [aimalz/prob-z#38]
This affects the internals of the code, but also the plotting.  As discussed on 2015-12-09 after group meeting.

**Comment by aimalz** [comment]
I have the "truth" from which the true redshifts are directly drawn as a smooth function.
![physpz](https://cloud.githubusercontent.com/assets/8606810/11754548/0a9617ce-a01a-11e5-8140-3126bdb01f81.png)
I plot this along with the binned true N(z) that is represented by a small number of parameters.
![truenz](https://cloud.githubusercontent.com/assets/8606810/11754562/1d48c4f2-a01a-11e5-9f1d-fe7d2a53cf66.png)
Is this what you're talking about?

**Comment by davidwhogg** [comment]
Drop True and call what you call "underlying" true.  And make sure that your true z values are drawn from that distribution.

**Comment by aimalz** [comment]
Cool, "true" has been changed to "sampled" and "underlying" has been changed back to "true" (as it was before you requested otherwise); of course the sampled N(z) will approach the underlying N(z) for large survey size.  The true redshifts are already drawn from the continuous true (formerly underlying) distribution.

**Comment by davidwhogg** [comment]
Drop "sampled".  If you were going to include it, it should be called "actual sample of z_true".  It is not a thing, from our perspective, so drop it.

Also, MLE should be changed to "maximum marginalized LF"

Also, MAP should be changed to "histogram of interim MAPs"

**Comment by davidwhogg** [comment]
By the way, I never requested that we change the meaning or name of "true".  I think that must be a misunderstanding.  There is only one possible meaning of "true", ever.  It is the PDF from which the redshifts were drawn.

**Comment by aimalz** [comment]
Okay, I probably wasn't clear about what I meant in distinguishing "true" from "sampled" in the first place.  I think this is closer to what you wanted:
![truenz](https://cloud.githubusercontent.com/assets/8606810/11769762/d5ee18d8-a1bc-11e5-8bc8-775929218220.png)
For the sake of keeping the legend small, I'll think of a shorthand for histogram of interim MAPs and explain it in the paper.

**Comment by davidwhogg** [comment]
No, I disagree -- a misleading caption is worse than no caption.  Go to long names.  Because MAP and MMLE have OTHER MEANINGS that we don't want to conflict with.  Trust me.
